README.md

```diff
@@ -1 +1,23 @@
 # api.cucumber.io
+
+This repo was created to store and publish generated API docs for Cucumber implementations.
+
+API docs tend to be big, and there are many versions. Keeping them in a separate repo means not bloating the main docs repo.
+
+For now, it should include generated docs per release (even though it could be useful to be able to refer to master branch docs also).
+
+The site is (currently) live at: https://hunter-ethel-13616.netlify.com/README.md
+
+
+## cucumber-jvm
+### 2.0.0
+[Javadocs for cucumber-jvm v 2.0.0](https://hunter-ethel-13616.netlify.com/cucumber-jvm/2.0.0/javadoc/)
```

Comment:
Use relative URLs: `/cucumber-jvm/2.0.0/javadoc/`

Reply:
Created and merged PR#4; just noticed however that the relative URL links to the github folder, while I meant to link to the live documentation...